# default transformation parameter

default_tf_param:
    x_trans: 0
    y_trans: 0
    z_trans: 0
    roll:    0
    pitch:   0
    yaw:     0
